docs/DeveloperGuide.md

```diff
@@ -1210,13 +1210,14 @@ testers are expected to do more *exploratory* testing.
 
```

Comment:
Perhaps just elaborate a bit more, i.e., "The prerequisites provided are specific to the example instructions provided."

Otherwise, I'm not sure if there might be some confusion that `list` **must** be run before every command and it doesn't work with `find`, `sort`, etc.

Reply:
have edited it such that find and sort does not have the prerequisite to run the `list` command in their use cases, so now not all commands have a `list` prereq